src/ImageSharp/Formats/Png/Filters/PaethFilter.cs

```diff
@@ -127,5 +180,70 @@ private static byte PaethPredictor(byte left, byte above, byte upperLeft)
 
             return upperLeft;
         }
+
+#if SUPPORTS_RUNTIME_INTRINSICS
+        private static Vector256<byte> PaethPredictor(Vector256<byte> left, Vector256<byte> above, Vector256<byte> upleft)
+        {
+            Vector256<byte> zero = Vector256<byte>.Zero;
+
+            // Here, we refactor pa = abs(p - left) = abs(left + above - upleft - left)
+            // to pa = abs(above - upleft).  Same deal for pb.
+            // Using saturated subtraction, if the result is negative, the output is zero.
+            // If we subtract in both directions and `or` the results, only one can be
+            // non-zero, so we end up with the absolute value.
+            Vector256<byte> sac = Avx2.SubtractSaturate(above, upleft);
+            Vector256<byte> sbc = Avx2.SubtractSaturate(left, upleft);
+            Vector256<byte> pa = Avx2.Or(Avx2.SubtractSaturate(upleft, above), sac);
+            Vector256<byte> pb = Avx2.Or(Avx2.SubtractSaturate(upleft, left), sbc);
+
+            // pc = abs(left + above - upleft - upleft), or abs(left - upleft + above - upleft).
```

Comment:
This level of code comments is lush!

Reply:
saucecontrol did a great job. Should contribution be documented somewhere?
Otherwise it's all fine right?

Reply:
> Should contribution be documented somewhere?

Not necessary at all.  I might have worded the comments a little differently if I knew they were going in as-is, but as long as someone other than me understands them, it's all good 😄

Reply:
Everything looks great! 

From a contribution documentation perspective, everything we do here is collaborative and conversation history is persistent so we don't add individual attribution.